metrics/storage/webtooling.sh

```diff
@@ -74,55 +131,78 @@ function main() {
 		exit 1
 	fi
 
-	containers=()
-	# Check tools/commands dependencies
-	local cmds=()
-	cmds+=("docker")
+	local i=0
+	local containers=()
+	local cmds=("docker")
+	local not_started_count=$NUM_CONTAINERS
 
 	restart_containerd_service
+	# Check tools/commands dependencies
+	clean
 	init_env
 	check_cmds "${cmds[@]}"
 	check_ctr_images "$IMAGE" "$DOCKERFILE"
-
 	metrics_json_init
-
 	save_config
 
+	info "Creating $NUM_CONTAINERS containers"
+
 	for ((i=1; i<= "$NUM_CONTAINERS"; i++)); do
-		containers+=($(random_name))
+		c_name="$(random_name)"
+		containers+=("$c_name")
```

Comment:
why have two variables ? why not just `c_name+="$(random_name)"` or just `containers+=($(random_name))`

Reply:
@GabyCT good recommendation to compact the code.